### PR TITLE
Adds multilayer pipe subtypes for mapping

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
code: Added multilayer subtypes for mapping.
/:cl:

[why]: It's a pain in the dong to map multilayer piping. This PR takes the var editing work out of it and should make it easy to do multilayer atmos setups.

### CODE REVIEW

File: code/modules/atmospherics/machinery/components/binary_devices/dp_vent_pump.dm [hunk 1]
```diff
@@ -31,21 +31,61 @@ Acts like a normal vent, but has an input AND output.
 	//EXT_BOUND: Do not pass external_pressure_bound
 	//INPUT_MIN: Do not pass input_pressure_min
 	//OUTPUT_MAX: Do not pass output_pressure_max
+	
+/obj/machinery/atmospherics/components/binary/dp_vent_pump/layer1
+	piping_layer = 1
+	pixel_x = -5
+	pixel_y = -5
```
Comment (ShizCalev) [inline code/modules/atmospherics/machinery/components/binary_devices/dp_vent_pump.dm]: make these all defines please

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by 81Denton** [comment]
@ShizCalev Done!

**Comment by Cyberboss** [comment]
If you don't mind not being able to visibly see the layers on the map, you could instead add a var called initial_layer on the common parent that sets all this up for you

**Comment by 81Denton** [comment]
Being unable to see the layer/pixel differences while map editing would make it pretty hard to maintain complex piping.

**Pull request merged**